listener.go

```diff
@@ -152,13 +153,13 @@ type listener struct {
 func (l *listener) WrapExpiration(days int) net.Listener {
 	ctx, cancel := context.WithCancel(context.Background())
 	go func() {
-		time.Sleep(5 * time.Minute)
+		time.Sleep(30 * time.Second)
 
 		for {
 			wait := 6 * time.Hour
-			if err := l.checkExpiration(days); err != nil {
-				logrus.Errorf("failed to check and refresh dynamic cert: %v", err)
-				wait = 5 + time.Minute
+			if err := l.checkExpiration(days); err != nil && err != cert.ErrStaticCert {
```

Comment:
Can you include a comment describing the state of when `l.checkExpiration` returns an error that IS `cert.ErrStaticCert`?

Reply:
I added a comment. Given that there's user action necessary to resolve this state, I think continuing to print errors to the log. We could also consider attaching an Event to the Kubernetes secret that the cert is stored in, but that's probably out of scope for resolving this issue.